푸터-마일드/layout/basic/css/ec-base-layer.css

```
@@ -0,0 +1,65 @@
+	.ec-base-layer.typeLayer { position:absolute; margin-top:10px; }
+    /* typeSide */
+    .ec-base-layer.typeSide { position:fixed; display: flex; flex-direction: column; height:100%; border: 0; }
+    .ec-base-layer.typeSide .header { padding: 74px 16px 10px; margin: 0; border: 0; }
+	.ec-base-layer.typeSide .header h2 { font-size: 18px; font-weight:normal; }
+	.ec-base-layer.typeSide .content { position: relative; padding: 0; flex: 1; overflow: auto; height:100%; }
+    .ec-base-layer.typeSide .ec-base-button[class*="gColumn"] {margin: 25px 16px; }
+    .ec-base-layer.typeSide .btnClose { top:5px; width:24px; height:24px; padding: 20px 17px; }
+    .ec-base-layer.typeSide .btnClose:before {right:12px; height:25px; }
+    .ec-base-layer.typeSide .btnClose:after {top:12px; width:25px;}
+    /* gAlert */
+    .ec-base-layer.gAlert > .header { display: block; overflow: hidden; position: absolute; font-size: 1px; line-height: 0; color: transparent; text-indent: -150%; white-space: nowrap; padding: 0; color: transparent; border: 0; }
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)